app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuEnhancedTrackingProtectionRobot.kt

```diff
@@ -96,13 +154,36 @@ private fun assertEnhancedTrackingProtectionDefaults() {
     )
 }
 
+private fun assertRadioButtonDefaults() {
+    onView(
+        allOf(
+            withId(org.mozilla.fenix.R.id.radio_button),
+            hasSibling(withText("Standard"))
```

Comment:
As mentioned before, if possible, let's try to get rid of hasSibiling and similar dependencies. Changing that for something like this also works:
```
onView(
        allOf(withText("Standard"))
    ).assertIsChecked(false)
```

Reply:
Ok